src/test/kotlin/uk/gov/justice/digital/hmpps/jobsboard/api/jobs/application/ExpressionOfInterestEditorShould.kt

```diff
@@ -0,0 +1,217 @@
+package uk.gov.justice.digital.hmpps.jobsboard.api.jobs.application
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.extension.ExtendWith
+import org.mockito.ArgumentMatchers.any
+import org.mockito.InjectMocks
+import org.mockito.Mock
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.kotlin.argumentCaptor
+import org.mockito.kotlin.never
+import org.mockito.kotlin.verify
+import org.mockito.kotlin.whenever
+import uk.gov.justice.digital.hmpps.jobsboard.api.commons.domain.EntityId
+import uk.gov.justice.digital.hmpps.jobsboard.api.jobs.domain.ExpressionOfInterest
+import uk.gov.justice.digital.hmpps.jobsboard.api.jobs.domain.ExpressionOfInterestId
+import uk.gov.justice.digital.hmpps.jobsboard.api.jobs.domain.ExpressionOfInterestRepository
+import uk.gov.justice.digital.hmpps.jobsboard.api.jobs.domain.Job
+import java.util.*
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFailsWith
+
+@ExtendWith(MockitoExtension::class)
+class ExpressionOfInterestEditorShould : TestBase() {
```

Comment:
Could this test file be divided in half if we split 'editor' into 'creator' and 'deleter'?

Reply:
sure, I will do it along with the spit

Reply:
updated